docs/getting-started-guide.org

```diff
@@ -125,61 +122,58 @@ occur. Let's start by creating the ~init.el~ file in your
 #+end_src
 
 Emacs customization is largely a matter of setting the values of certain
-variables. The Emacs customization system will store these in the file
-known as the ~custom-file~ which defaults to your initialization file, in
-this case called ~init.el~. We'd rather keep that separate from what we are
-doing in the ~init.el~ file as that code is auto-generated by Emacs. In a
-sense, it is a duplication of the work we will be doing as the values we
-set programmatically will also be stored in this file and loaded by the
-code above.
+variables. The Emacs customization system will store these in the file known as
+the ~custom-file~ which defaults to your initialization file, in this case called
+=init.el=. We'd rather keep that separate from what we are doing in the =init.el=
+file as that code is auto-generated by Emacs. In a sense, it is a duplication of
+the work we will be doing as the values we set programmatically will also be
+stored in this file and loaded by the code above.
 
 For more information on the ~custom-file~ see [[info:emacs#Saving Customizations][Saving Customizations]],
 or on the web: [[https://www.gnu.org/software/emacs/manual/html_node/emacs/Saving-Customizations.html][Saving Customizations]].
 
 Right below the ~custom-file~ setup, add the following line (which assumes
-the ~crafted-emacs-home~ is the ~crafted-emacs~ directory found in your
+the ~crafted-emacs-home~ is the =crafted-emacs= directory found in your
 home folder):
 
 #+begin_src emacs-lisp
   (load "~/crafted-emacs/modules/crafted-init-config")
 #+end_src
 
 In a rough outline, the ~crafted-init-config~:
-- sets up the load-path, so code from ~modules~ and ~custom-modules~ can
+- sets up the load-path, so code from =modules= and =custom-modules= can
   be loaded with ~require~
 - sets up the ~info~ system to make this documentation accessible from
   within Emacs' info buffer
 - ensures all customizations and packages are saved to the ~custom-file~
 
-These lines define the top of our ~init.el~, everything else will be put
+These lines define the top of our =init.el=, everything else will be put
 below.
 
 * Crafted Emacs Modules
 
-Now that you have bootstrapped @@texinfo:@emph{Crafted Emacs}@@,
+Now that you have bootstrapped Crafted Emacs,
 you can start configuring Emacs using Crafted Emacs Modules.
 
 ** Installing packages
 
-The standard approach to finding and installing packages is to use
-the command ~M-x list-packages RET~, which will bring up a
-user interface to search for packages, review the package details,
-install, update or remove package.
-For more information: [[info:emacs#Packages][Packages]] or on the web: [[https://www.gnu.org/software/emacs/manual/html_node/emacs/Packages.html][Packages]].
+The standard approach to finding and installing packages is to use the command
+=M-x= ~list-packages RET~, which will bring up a user interface to search for
```

Comment:
```suggestion
~M-x list-packages RET~, which will bring up a user interface to search for
```

This one is awkward.  It renders in the info file a `M-x` `list-packages RET`, but it seems like it should render as `M-x list-packages RET`.  Maybe this should get the `~` treatment?  In the GitHub rendering, there is a space between the `M-x` and the `install-packages RET`.  I'd prefer there not be that space as this is a single command.

Reply:
I agree, I think personally this falls under both "keystrokes or keybindings" and "text (to be) entered into the minibuffer", so `=`, but I'm ok with `~` as well.

Reply:
I agree it can't stay like this. I'm also leaning towards `=`. And, come to think of it, we can get rid of the `RET` in this case, right?

Reply:
When I went through the docs to look for other occurrences of `=M-x= ~foo~`, I saw that we already had several occurrences of `=M-x foo=` already. I changed all others to match them, hope that's OK.
Thanks!

Reply:
> I agree, I think personally this falls under both "keystrokes or keybindings" and "text (to be) entered into the minibuffer", so `=`, but I'm ok with `~` as well.

Ah... Yes, that works for me (agree with `=`).